README.md

```diff
@@ -5,52 +5,63 @@
 
 ## Description
 
-This repository will demostrate how to use [VUDRM](https://vudrm.vualto.com/) with the [JW Player](https://jwplayer.com).
-If you have any questions please contact <support@vualto.com>
+#### !!!!TODO!!!!: Change the below to the JW Studio DRM docs 
+This repository will demonstrate how to use [Studio DRM](https://jw-studio-drm-docs) with [JW Player](https://jwplayer.com).
+If you have any questions please contact <support@jwplayer.com>

```

Comment:
Why are you surrounding this with `<>`?
Are trying to achieve this?
```suggestion
If you have any questions please contact [support@jwplayer.com](support@jwplayer.com).

```

Reply:
The <> does the same thing as your suggested change.